gui/src/app/SamplerOutputView/SummaryView.tsx

```diff
@@ -1,5 +1,6 @@
 import { FunctionComponent, useMemo } from "react"
 import { computeMean, computePercentile, computeStdDev } from "./util"
+import { compute_effective_sample_size, compute_split_potential_scale_reduction } from "./stan_stats/stan_stats"
```

Comment:
This comment didn't post with the others - could we move the code currently in `./util` into this folder and then de-duplicate it? I think we have two mean calculations now, for example

Reply:
I'd like to keep stan_stats as self-contained as possible and exposing as minimum as possible, because we're planning to make this a separate package (I think it would be named something other than stan_stats of course)

Reply:
I was imagining said other package would also include the quantiles, mean, and standard deviation functions. I guess it doesn’t need to, but they’re related and nice to have somewhere

Reply:
I see. Yeah I think that would be nice... but would require some planning.